bin/nvram-compact

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/perl
+# Remove invalid and non-data entries from an NVRAM section
+# This only handles the common cases
+
+use strict;
+use warnings;
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use EFI;
+use Getopt::Long;
+use File::Basename;
+use File::Temp 'tempfile';
+use Data::Dumper;
+
+my $nvram_guid = "cef5b9a3-476d-497f-9fdc-e98143e0422c";
+my $verbose = 0;
+
+local $/;
+my $ffs = <>;
```

Comment:
So the assumption is that only one nvram ffs is processed each time? a simple usage string would be helpful